src/app/core/components/left-search-result/left-search-result.component.html

```diff
@@ -1,17 +1,48 @@
-<a class="alg-link base-color" (click)="close.emit()">(X) close</a>
+<div class="container" (click)="close.emit()" *ngIf="data !== undefined">
+  <div class="back-button-container cursor-pointer">
+    <button
+      class="alg-button-icon back-button rounded"
+      icon="ph ph-arrow-left"
+      pButton
+    ></button>
+    <p class="back-button-title" i18n>
+      {{ data.length }}
+      {{ data.length | i18nPlural: {
+        '=1': 'activity found',
+        'other': 'activities found'
```

Comment:
In French, the plural is for >1 (so 0 is singular). It should be supported then (having 3 cases if i18n does not know the per-language differences)